### PR TITLE
fixed typos and added link

### PR DESCRIPTION
**Pull request by aaronhmiller**
doing this the right way :)

### CODE REVIEW

File: app/plugins/ee-openid-connect.md [hunk 1]
```diff
@@ -63,24 +63,21 @@ The plugin has been tested with several OpenID Connect capable providers such as
...
 - [Paypal][paypal]
 - [PingFederate][pingfederate]
-- [Salesfoce][salesforce]
+- [Salesforce][salesforce]
 - [Yahoo!][yahoo]
 
-As long as your provider supports OpenID Connect standards the plugin should
-work, even if it is not specifically tested against it. Let us know if you
-want your provider to be tested and added to the list.
+As long as your provider supports OpenID Connect standards (exposing a {service_url}.well-known/openid-configuration endpoint) the plugin should work, even if it is not specifically tested against it. Let us know if you want your provider to be tested and added to the list.
```
Comment (thibaultcha) [inline app/plugins/ee-openid-connect.md]: Should the link here be enclosed in code blocks for readability?

(exposing a `{service_url}.well-known/openid-configuration` endpoint)

File: app/plugins/ee-openid-connect.md [hunk 2]
```diff
@@ -63,24 +63,21 @@ The plugin has been tested with several OpenID Connect capable providers such as
...
 
 <br />
 
 <div class="alert alert-warning">
-  <strong>Enterprise-Only</strong> This plugin is only available with an
-  Enterprise Subscription.
```
Comment (thibaultcha) [inline app/plugins/ee-openid-connect.md]: This change is undesired - it is there to respect the 80 characters-wide limit per line.

File: app/plugins/ee-openid-connect.md (hunk 1 above)
Comment (thibaultcha) [reply app/plugins/ee-openid-connect.md]: We should also respect the 80 chars limit @bungle tried to enforce here... Tons of plugins will do this automatically for various text editors.

Comment (aaronhmiller) [reply app/plugins/ee-openid-connect.md]: fixing and will re-submit

File: app/plugins/ee-openid-connect.md (hunk 2 above)
Comment (aaronhmiller) [reply app/plugins/ee-openid-connect.md]: fixing and will re-submit

**Comment by aaronhmiller** [comment]
will re-submit in a new PR

**Comment by thibaultcha** [comment]
@aaronhmiller You can keep the same PR open but add new commits or amend the commit(s) already proposed.

**Comment by thibaultcha** [comment]
It It is preferred compared to closing and opening new PRs for each round of review...

**Pull request closed without merging**